### DISCUSSION TITLE
Maintenance - dusting off the mirror

### DISCUSSION THREAD

**Discussion by Hazoooka** [sn4k3/UVtools#901] [General]
Hi everyone.

Those of you not accustomed with the problem I want to show how the mirror condition inside our printers can impact the print.
Printer used rather seldom way.
Just cleaned it recently, after less than a year, see the pictures attached.
They speak for themselves, no further explanation necessary.

Keep the mirror clean.

Regards,
Witold.
![#1 before](https://github.com/user-attachments/assets/916e145d-279b-48de-949b-723b53a99d7f)
![#2 after](https://github.com/user-attachments/assets/0e71483c-4508-4a9a-a1c6-1965f9e1a49f)

**Comment by sn4k3** [comment]
Dust particles on mirror and lens will reflect light differently, depending on the amount can also create shadows and will change the quality of the light.

Printers have a open design with fans for airflow, that will attract dust inside overtime.
Have printers in a clean environment is important due this. A garage full of dust can impact the print quality in a first run...

Further notes:

- Do not rub mirror, use almost no force for the first pass and dry clean with a lint free cloth. (Smash clean the mirror can lead dust particles to scratch mirror so it's important to get them away lightly)
- Do not use any chemicals to clean the mirror. If you never touch the mirror with your fingers the chances are you won't need any solution. But water is just fine.
- (This notes are also valid for your TV and laptop non-touch screen)

**Reply by Hazoooka** [reply]
True.
For wiping the mirror a dry silk cloth is the best.

Also it's good to dust off the LED set at the same time which I haven't mentioned, but it's pretty obvious.

**Comment by Hazoooka** [comment]
Regarding the impact, it does impact the prints, it's causing the delimitation of the base layers.
Dust off the printer and it's going back to normal.
